lib/hyrax/configuration.rb

```diff
@@ -377,6 +377,26 @@ def display_media_download_link?
     # @!endgroup
     # @!group Valkyrie
 
+    ##
+    # @return [Valkyrie::StorageAdapter]
+    def branding_storage_adapter
+      @branding_storage_adapter ||=
+        begin
+          Valkyrie::StorageAdapter.register(
```

Comment:
i guess this should probably move to an initailizer?

Reply:
Yeah, that looks like the better pattern.

Reply:
how does this version look?